airflow/www/templates/airflow/dag_code.html

```diff
@@ -32,23 +32,8 @@
 {% block tail_js %}
   {{ super() }}
   <script>
-    function toggleWrap() {
-      $('.code pre').toggleClass('wrap')
-    };
-
-    // We blur task_ids in demo mode
-    $(document).ready(function() {
-      if ("{{ demo_mode }}" == "True") {
-        $("pre span.s").css({
-          'text-shadow': '0 0 10px red',
-          'color': 'transparent',
-        });
-      }
-    });
-
-    // pygments generates the HTML so set wrap toggle via js
-    if ("{{ wrapped }}" == "True") {
-      toggleWrap();
-    };
+    const isDemoMode = '{{ demo_mode }}' === 'True';
```

Comment:
Ideally, we should manage to delete all inline scripts. For this purpose, we can pass this data as the page metadata.
```javascript
function getMetaValue(name) {
  const el = document.querySelector('meta[name="' + name + '"]')
  if (!el) {
    return
  }
  return el.getAttribute("content")
}

global.getMetaValue = getMetaValue
```
```
<meta name="dag-timezone" content="{{ dag.timezone.name }}">
```
```
var dag_timezone = gettMetaValue("dag-timezone")
```

Reply:
Good idea. Updated,